backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5575090_sys_gh10412AddMessageAndTranslations.sql

```diff
@@ -0,0 +1,35 @@
+-- 2020-12-16T13:17:59.005Z
+-- URL zum Konzept
+INSERT INTO AD_Message (AD_Client_ID,AD_Message_ID,AD_Org_ID,Created,CreatedBy,EntityType,IsActive,MsgText,MsgType,Updated,UpdatedBy,Value) VALUES (0,545019,0,TO_TIMESTAMP('2020-12-16 10:17:56','YYYY-MM-DD HH24:MI:SS'),100,'U','Y','Cannot add product due haddex check','E',TO_TIMESTAMP('2020-12-16 10:17:56','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.order.producthadexerror1')
+;
+
+-- 2020-12-16T13:17:59.234Z
+-- URL zum Konzept
+INSERT INTO AD_Message_Trl (AD_Language,AD_Message_ID, MsgText,MsgTip, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language, t.AD_Message_ID, t.MsgText,t.MsgTip, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Message t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y') AND t.AD_Message_ID=545019 AND NOT EXISTS (SELECT 1 FROM AD_Message_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Message_ID=t.AD_Message_ID)
+;
+
+-- 2020-12-16T13:19:25.696Z
+-- URL zum Konzept
+UPDATE AD_Message SET Value='de.metas.order.producthadexerror',Updated=TO_TIMESTAMP('2020-12-16 10:19:25','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Message_ID=545019
```

Comment:
See backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5574550_sys_gh10412HaddexColumnsToProductTableAndSysConfigsAndErrMsgAndProductWindowAdjustments.sql.

Dragos already added an AD_Message with Value=`de.metas.base.producthadexerror`.

I prefer your naming, i.e. `de.metas.order.producthadexerror` but in this case we have to remove the AD_Message from previous script.

Else we will add an unused AD_Message record which would make harder to maintain.